### DISCUSSION TITLE
Avoiding dirt roads

### DISCUSSION THREAD

**Discussion by wavard** [devemux86/cruiser#109] [Q&A] [answered]
I can't find a setting to avoid dirt roads. Is it possible?

**Comment by devemux86** [comment, marked as answer]
Hi @wavard,

The available avoid options depend on what the routing services can provide.

Currently BRouter (offline) can offer such options, if we customize its profiles.

(I can check again what is available in the online routing services)

**Comment by wavard** [comment]
Yes please.

Regards

Walter
